src/gcodeExport.cpp

```diff
@@ -377,14 +377,27 @@ double GCodeExport::getTotalFilamentUsed(int extruder_nr)
     return extruder_attr[extruder_nr].totalFilament;
 }
 
-double GCodeExport::getTotalPrintTime()
+std::vector<double> GCodeExport::getTotalPrintTimes()
```

Comment:
The CuraEngine side of this PR looks good to me!

Just one little comment: I'd like to rename this function to `getTotalPrintTimePerFeature` so as to be more specific.